src/routes.js

```diff
@@ -316,3 +315,27 @@ export default (apollo, store) => {
     }
   ]
 }
+
+// handle direct sign in from the extension via deeplink
+export async function extensionSignIn({ to, next }, apollo, store) {
+  const {
+    data: { network }
+  } = await apollo.query({
+    query: gql`
+      query Network {
+        network(id: "${to.params.network}") {
+          id
+          slug
```

Comment:
Why are we not sending here `network_type` as well?

Reply:
not needed here, is it?

Reply:
to tell the extension the network type that need to sign on it, no? I guess in the extension we will also introduce this `getCosmosSigner` & `getWhateverNetworkSigner` logic? or I got that wrong?